service-tests/bundlephobia.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('./runner/service-tester')
+
+const t = new ServiceTester({
+  id: 'bundlephobia', title: 'NPM package bundle size',
+})
+
+const formats = {
+  A: '/bundlephobia/:type/:package.:format',
+  B: '/bundlephobia/:type/:package/:version.:format',
+  C: '/bundlephobia/:type/@:scope/:package.:format',
+  D: '/bundlephobia/:type/@:scope/:package/:version.:format',
+}
+
+const withoutErrorsMin = {
+  A: '/min/preact.json',
+  B: '/min/preact/8.2.7.json',
+  C: '/min/@cycle/core.json',
+  D: '/min/@cycle/core/7.0.0.json',
+}
+
+const withoutErrorsGzip = {
+  A: '/gzip/preact.json',
+  B: '/gzip/preact/8.2.7.json',
+  C: '/gzip/@cycle/core.json',
+  D: '/gzip/@cycle/core/7.0.0.json',
+}
+
+const withoutErrorSizesMin = {
+  A: Joi.string().regex(/^\d+[.]?\d+ kB$/),
+  B: '8.94 kB',
+  C: Joi.string().regex(/^\d+[.]?\d+ kB$/),
+  D: '3.51 kB',
+}
+
+const withoutErrorSizesGzip = {
+  A: Joi.string().regex(/^\d+[.]?\d+ kB$/),
+  B: '3.58 kB',
+  C: Joi.string().regex(/^\d+[.]?\d+ kB$/),
```

Comment:
You could use [`isFileSize`](https://github.com/badges/shields/blob/master/service-tests/helpers/validators.js#L53) for this.